extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_index.php

```diff
@@ -0,0 +1,74 @@
+<div class="wds-global-navigation-wrapper">
+	<div id="globalNavigation" class="wds-global-navigation <?= isset( $model['fandom_overview'] ) ? '' : ' wds-search-is-always-visible'; ?>">
+		<div class="wds-global-navigation__content-bar">
+			<?= $app->renderView( 'DesignSystemGlobalNavigationService', 'logo', [ 'model' => $model['logo'] ] ); ?>
+			<?php
+			if ( isset( $model['fandom_overview'] ) ):
+				foreach ( $model['fandom_overview']['links'] as $link ):
+			?>
+					<?= $app->renderView( 'DesignSystemGlobalNavigationService', 'linkBranded', [ 'model' => $link ] ); ?>
+			<?php
+				endforeach;
+			endif;
+			?>
+			<?= $app->renderView(
+				'DesignSystemGlobalNavigationService',
+				'links',
+				[
+					'model' => $model['wikis'],
+					'type' => 'wikis-menu'
+				]
+			); ?>
+			<form class="wds-global-navigation__search" action="<?= Sanitizer::encodeAttribute( $model['search']['module']['results']['url'] ); ?>">
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "search" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "module" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)